packages/discussions/test/reactions.test.ts

```diff
@@ -0,0 +1,50 @@
+import { createReaction, deleteReaction } from "../src/reactions";
+import * as req from "../src/request";
+import { PostReaction } from "../src/types";
+
+describe("reactions", () => {
+  let requestSpy: any;
+  const response = new Response("ok", { status: 200 });
+
+  beforeEach(() => {
+    requestSpy = spyOn(req, "request").and.returnValue(
+      new Promise(resolve => {
+        resolve(response);
+      })
```

Comment:
There's > 5k loc in this PR, this can save 2 of them:

```suggestion
      Promise.resolve(response);
```